.travis.yml

```diff
@@ -1,5 +1,6 @@
 language: java
 sudo: false
+dist: trusty
```

Comment:
Why do we go for this instead of changing it to openjdk8?

Reply:
Just changing openjdk was not enough as other changes were needed to work with Xenial. I did two of those changes and then got stuck, so figured reverting to trusty would serve just as well for now